src/main/java/karuna/karuna_backend/exception/keys/FailedValidationErrorKey.java

```diff
@@ -0,0 +1,13 @@
+package karuna.karuna_backend.exception.keys;
+
+//TODO: SWAGGER SCHEMA AFTER DISCUSS ABOUT NAMES OF KEYS
+public enum FailedValidationErrorKey {
+    SIZE,
+    PATTERN,
+    EMAIL,
+    MAX,
+    MIN,
+    NOT_NULL,
+    NOT_BLANK,
+    UNKNOWN
+}
```

Comment:
Implement a little bit more detailed keyName. After that obviously add swagger schema.

Reply:
INVALID_PATTERN
INVALID_EMAIL_PATTERN
VALUE_TOO_LONG
VALUE_TOO_SHORT
VALUE_IS_NULL
VALE_IS_BLANK

And just describe them that should do the job for now

Reply:
my is ok, or i should change?

Reply:
i fix before u comment

Reply:
Okay i fixed like u said